common/src/main/java/io/homeassistant/companion/android/common/sensors/DisplaySensorManager.kt

```diff
@@ -98,4 +159,110 @@ class DisplaySensorManager : SensorManager {
             mapOf()
         )
     }
+
+    private fun updateScreenOrientation(context: Context) {
+        if (!isEnabled(context, screenOrientation)) {
+            return
+        }
+
+        @Suppress("DEPRECATION")
+        val orientation = when (context.resources.configuration.orientation) {
+            Configuration.ORIENTATION_PORTRAIT -> "portrait"
+            Configuration.ORIENTATION_LANDSCAPE -> "landscape"
+            Configuration.ORIENTATION_SQUARE -> "square"
+            Configuration.ORIENTATION_UNDEFINED -> STATE_UNKNOWN
+            else -> STATE_UNKNOWN
+        }

```

Comment:
I believe we should add these possible states as an `options` attribute to help with automation editing